### DISCUSSION TITLE
Airflow logs not found giving error-The task might not have been executed or worker executing it might have finished abnormally (e.g. was evicted).

### DISCUSSION THREAD

**Discussion by Shifa15** [apache/airflow#34508] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Airflow version is 2.5.3
When the DAGs are executing at scheduled time it is giving the following error in 1 or 2 attempts and then executing the task.
```
Log file is not found: 
*** The task might not have been executed or worker executing it might have finished abnormally (e.g. was evicted).
*** Please, refer to https://cloud.google.com/composer/docs/how-to/using/troubleshooting-dags#common_issues hints to learn what might be possible reasons for a missing log.
```

### What you think should happen instead

There might be following possible cause
- logs clean up required
- pod has stopped working
- worker is executing the task

### How to reproduce

Not sure

### Operating System

Windows

### Deployment

Google Cloud Composer

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by hussein-awala** [comment]
Which executor are you using? Did you try to reproduce it with the latest version?

**Reply by Taragolis** [reply]
I almost sure this is Google Cloud Composer and the error it is not a part of Apache Airflow, at least I can't find it.

@Shifa15 I would recommend reach Google Cloud support because it managed Airflow service and OSS Community in general don't know which changes Google made in their implementation 😞 .

If you would lucky and receive information how to reproduce it, then someone from Airflow community might potentially check is this persist or not.

**Comment by aenni0409** [comment]
Hi @Shifa15  - 
Do you get any updates regarding this? Also got the same error after upgrading to 2.5.3 with GCP composer 2.4.3. 
We have reached out Google support but wonder if you have any updates you can share. Thank you!
